mappings/net/minecraft/world/biome/layer/ChooseBiomesLayer.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS cgt net/minecraft/world/biome/layer/ChooseBiomesLayer
+	FIELD a BIRCH_ID I
+	FIELD b DESERT_ID I
+	FIELD c MOUNTAINS_ID I
+	FIELD d FOREST_ID I
+	FIELD e SNOWY_TUNDRA_ID I
+	FIELD f JUNGLE_ID I
+	FIELD g BADLANDS_PLATEAU_ID I
+	FIELD h WOODED_BADLANDS_PLATEAU_ID I
+	FIELD i MUSHROOM_FIELDS_ID I
+	FIELD j PLAINS_ID I
+	FIELD k GIANT_TREE_TAIGA_ID I
+	FIELD l DARK_FOREST_ID I
+	FIELD m SAVANNA_ID I
+	FIELD n SWAMP_ID I
+	FIELD o TAIGA_ID I
+	FIELD p SNOWY_TAIGA_ID I
+	FIELD q DESERT_FOREST_MOUNTAINS_SWAMP_PLAINS_TAIGA [I
```

Comment:
Shouldn't these have a better name than just a list of the biomes the array contains (maybe look at the usages)?